core/server/data/meta/title.js

```diff
@@ -12,7 +12,7 @@ function getTitle(data, root, options) {
     options = options ? options : {};
 
     if (pagination && pagination.total > 1) {
-        pageString = ' (Page ' + pagination.page + ')';
+        pageString = _.has(options.hash, 'pageString') ? options.hash.pageString.replace('%', pagination.page) : ' (Page ' + pagination.page + ')';
```

Comment:
@ErisDS 

`{{meta_title pageString=(t...)}}`

Design Approval for translating the meta title if paginated.

Reply:
@mantebridts We had an internal chat about this.

`{{meta_title page=(t...)}}`

This is the version you've used in the first commit already - so your initial idea was for now the best.

Reply:
OK, so I revert this last commit and will change it back to 'page'!